### PR TITLE
Rearranged toolbar to be more like Lollypop (and more intuitive)

### PR DESCRIPTION
**Pull request by p3732**
Also made stop and volume optional.

### CODE REVIEW

File: src/headerbarui.c [hunk 1]
```diff
@@ -606,6 +644,8 @@ static const char settings_dlg[] =
     "property \"Use combined play/pause button\" checkbox headerbarui.combined_playpause 1;\n"
     "property \"Show seekbar\" checkbox headerbarui.show_seek_bar 1;\n"
     "property \"Hide seekbar on streaming\" checkbox headerbarui.hide_seekbar_on_streaming 0;\n"
+    "property \"Show stop button\" checkbox headerbarui.show_stop_button 1;\n"
```
Comment (saivert) [inline src/headerbarui.c:642]: Default values here don't match code.

**Review by saivert — COMMENTED** [review]
Is the title text supposed to be right aligned next to the seekbar?
It also doesn't steal enough space for itself when there is clearly room. Seems like seekbar has priority. Try to maximize deadbeef on a bigger monitor. Maybe you could look at this.

Also fix up the preferences (see comment).

**Comment by p3732** [comment]
> Is  the title text supposed to be right aligned next to the seekbar?

Indeed this troubled me as well, but the title text is set by deadbeef as a normal GTK headerbar text and those get aligned towards center by default. I don't really know how to approach this problem. Maybe looking into Gtk::HeaderBar::get_custom_title() will help.

>Seems like seekbar has priority.

Yes, that was intentional. However, I agree in retrospect it looks a bit weird. However I don't think the title should be as big as the seekbar at certain sizes. Instead of a proportion of 50%:50%, I would suggest something like 40%:60%. What do you think?

> Also fix up the preferences (see comment).

Noted.

**Comment by saivert** [comment]
Lollypop hides the standard titletext area of headerbar (which by default shows the windows title) and uses a custom widget for albumart and title. If you're up for it feel free to implement this in a separate PR.

**Pull request merged**